src/computation.ts

```diff
@@ -260,11 +271,12 @@ export class Computation {
           func.name,
         ),
       this.verbose,
+      this.spanContext,
```

Comment:
I think this needs to be `ctx.spanContext`

Reply:
Thanks! This solves the graph issue we were seeing